README.md

```diff
@@ -115,9 +115,12 @@ Now you can enable `pasteImage.showFilePathConfirmInputBox` to modify file path
 
 - `pasteImage.insertPattern`
 
-    The pattern of string that would be pasted to text. 
-
-    You can use variable:
+    The pattern of string that would be pasted to text.
+    
+    You can configure both the alt text and the file path.
+    For example: `![${imageFileNameWithoutExt}](${imageFilePath})`.
```

Comment:
Here's the added documentation

Reply:
it would be better to add the explanation of `${imageFileName}` and `${imageFileNameWithoutExt}` to variables  list

Reply:
I added a description of the example, if that's what you mean. :) The variable list below that line also describes those two variables.

Reply:
Oh, I've added it, I forgot.